editor/test/integration/test_util.clj

```diff
@@ -604,14 +610,6 @@
             (g/undo! graph-id)
             (recur (g/undo-stack-count graph-id))))))))
 
-(defn- make-app-view-select-fn [app-view]
-  (assert (g/node-id? app-view))
-  (let [selection-volatile (volatile! nil)
-        select-fn (fn select-fn [node-ids]
-                    (vreset! selection-volatile node-ids)
-                    (app-view/select app-view node-ids))]
-    [select-fn selection-volatile]))
-
```

Comment:
The `test-util/add-embedded-game-object!`, `test-util/add-embedded-component!` and `test-util/add-referenced-component!` functions no longer take an `app-view` argument, and make no attempts to update the selection in the `app-view`.